src/drawings.jl

```diff
@@ -642,7 +793,24 @@ function finish()
     Cairo.destroy(current_surface())
 
     if current_filename() != ""
-        write(current_filename(), current_bufferdata())
+        # next function call adresses the issue in
```

Comment:
This will be true for PNG, EPS, and PDF as well - why not make a special SVG route, and leave the other formats as is?

Reply:
confirmed and done